egui/src/widgets/text_edit/builder.rs

```diff
@@ -1091,11 +1091,31 @@ fn on_key_press(
             None
         }
 
+        Key::P | Key::N | Key::B | Key::F | Key::A | Key::E
+            if cfg!(target_os = "macos") && modifiers.ctrl && !modifiers.shift =>
```

Comment:
Note that `cfg!(target_os = "macos")` won't work when compiling to `wasm` (i.e. for web).

Perhaps we should add some platform-detection in `egui_web` so we can add `if ctx.options().is_mac` or similar?

Having it only work on native for now might be OK though.

Reply:
I'll take a note and might want to also implement this for web at some point, but for now I think native is ok.